apps/ejabberd/src/ejabberd_c2s.erl

```diff
@@ -2002,18 +2002,6 @@ presence_broadcast_first(From, StateData, Packet) ->
             StateData#state{pres_a = As}
     end.
 
-
--spec remove_element(E :: 'error' | tuple(),
-                     Set :: gb_set()) -> gb_set().
-remove_element(E, Set) ->
-    case ?SETS:is_element(E, Set) of
```

Comment:
But what's the point? The `gb_sets:del_element/2` will not crash if the removing element is not in the set. I don't think this check is necessary here.
Also `sets` module will cope with this. And if we are to use other set implementation this will be followed by tests.
BTW do we have an explicit test where we remove element from a set, but the element doesn't exist there?